suggested-study.md

```diff
@@ -86,6 +87,7 @@
 - [html-css-git-exercises](https://github.com/hackyourfuturebelgium/html-css-git-exercises)
```

Comment:
This link is dead, remove it?

Reply:
Update the link
https://internetingishard.netlify.app/html-and-css/

Reply:
+1